refine a little things of home page.

dqh725 opened the pull request:
https://www.pivotaltracker.com/n/projects/1108894/stories/107093106

just refine the home page with a few css changes, and add the mails request input field for add user emails to mailchimp subscriber list. 

As for backend action(probably a seperate controller) for to add email to mailchimp subscriber list: I am thinking use the gem of gibbon and create a controller for that, may be able to work this task next Tuesday. But i think Tom mighe want me off the backend.

dqh725 committed:
finish the first two task, the 3rd task need more backend support

dqh725 committed:
finish the first two task, the 3rd task need more backend support

zero51 commented:
For UI work, can you please add screenshots to the PR?

zero51 commented on app/views/home/_email_request.html.haml:
```diff
@@ -0,0 +1,12 @@
+
+
+%form.center-block.form-inline.search-form{role: "form", action: "#", method: "get"}
+    .form-group#email-field
```
Can we try and make sure that tabs are set to two spaces, across all file types?

zero51 commented on app/views/home/_aus_map.html.slim:
```diff
@@ -1,6 +1,6 @@
+    	h2.text-center Australia's Most Trusted Pet Care Community
     	.col-xs-12
-    		= image_tag 'home/aus-map.png', class: 'center-block img-responsive', style: 'height:300px'
+    		= image_tag 'home/australia.png', class: 'center-block img-responsive', style: 'height:300px; padding-bottom:15px;'
```
Tab size!

zero51 committed:
Remove attempts to deliver unsupported emails

https://www.pivotaltracker.com/story/show/104594914

In the move to Mandrill templates, a lot of system emails were
removed, one of which was UserMailer.update_account, which was
called after a successful update of a user's details. The app no
longer attemps deliver of this email. Calls to five other stale
emails were also removed.

adimasuhid committed:
Merge pull request #32 from pethomestay/fix/ad-95931740

fix/ad-95931740

zero51 committed:
Move to Capistrano 3 and Digital Ocean

https://www.pivotaltracker.com/story/show/107094090

zero51 committed:
Add AppSignal

zero51 committed:
Tweak deploy

zero51 committed:
Share local_env.yml

zero51 committed:
Remove Raygun and New Relic.

zero51 committed:
Link .rbenv-vars

zero51 committed:
Tag AppSignal errors so that they include basic user info

https://www.pivotaltracker.com/story/show/108134658

zero51 committed:
Fix feedback creation

https://www.pivotaltracker.com/story/show/108134712

* 'Validation failed: Rating is not included in the list'.
* Rating passed as string in params. Just removed the validation.

zero51 committed:
Intercept notifications in staging and development

https://www.pivotaltracker.com/story/show/107791794

In production and test envs, all notifications go through as normal.
In other envs, notifications go through if the user has a PHS email.
If the user has a different email address, then email notifications
are forwarded to dave@pethomestay.com and SMS notifications are
forwarded to 0416168436.

zero51 committed:
Missing mail interceptor initialiser

zero51 committed:
Absolutely position homepage navbar, set min height on banner

zero51 committed:
Fix horizontal alignement of press icons

zero51 committed:
If we must use title case, make sure it's consistent

zero51 committed:
Desktop homepage tidied up...

zero51 committed:
Cleanup responsive versions

zero51 committed:
Fix conflicts
